clowder/init_ephemeral_cluster.sh

```diff
@@ -3,11 +3,17 @@
 # See step 7 of this link: https://clouddot.pages.redhat.com/docs/dev/getting-started/ephemeral/onboarding.html
 echo "Loggin to Openshift cluster"
 sh ~/ephemeral-login.sh
+echo "Checking for reserved namespace"
+export NAMESPACE=$(bonfire namespace list --mine | grep "ephemeral" | awk '{print $1}' | xargs -n1)
 
-echo "Reserving namespace for 2 hours"
-export NAMESPACE=$(bonfire namespace reserve -d 2)
-
-echo "Namespace $NAMESPACE reserved"
+if [[ $NAMESPACE == *"ephemeral"* ]]; then
+    echo "Namespace $NAMESPACE reserved, extending for 2 hours"    
+    bonfire namespace reserve -d 2 $NAMESPACE 
```

Comment:
I guess that bonfire prolongs the reservation if it detects that it is under 2 hours, If I'm not mistaken.